_studio/shared/umc/codec/h264_dec/src/umc_h264_va_packer.cpp

```diff
@@ -426,7 +426,7 @@ void PackerVA::CreateSliceDataBuffer(H264DecoderFrameInfo * pSliceInfo)
         size += NalUnitSize;
     }
 
-    uint32_t const AlignedNalUnitSize = mfx::align_value<uint32_t>(size, 128);
+    uint32_t const AlignedNalUnitSize = mfx::align2_value<uint32_t>(size, 128);
```

Comment:
Do we still need explicit `uint32_t` here?

Reply:
no, we don't. I missed this one in cleanup commit. Will update